Main/Source/char.cpp

```diff
@@ -4498,7 +4500,9 @@ void character::DrawPanel(truth AnimationDraw) const
   PrintAttribute("Wis", WISDOM, PanelPosX, PanelPosY++);
   PrintAttribute("Wil", WILL_POWER, PanelPosX, PanelPosY++);
   PrintAttribute("Cha", CHARISMA, PanelPosX, PanelPosY++);
-  FONT->Printf(DOUBLE_BUFFER, v2(PanelPosX, PanelPosY++ * 10), WHITE, "Siz  %d", GetSize());
+  FONT->Printf(DOUBLE_BUFFER, v2(PanelPosX, PanelPosY++ * 10), WHITE, "Size %d cm", GetSize());
+  FONT->Printf(DOUBLE_BUFFER, v2(PanelPosX, PanelPosY++ * 10), WHITE, "Wght %d kg", GetTotalCharacterWeight());
```

Comment:
That's unfortunately impossible - full "weight" plus the numbers will not fit on one line and will go off screen. I used "wght" as a compromise. I also considered "Mass" and "W.", but thought this shortening was the best as it fits nicely with (also 4 lettered) "Size" and does not sound as weird and confusing as "Mass".

Reply:
Ah damn, this is hard. I tried "Wgt" and it looks pretty ok I guess, especially if we keep "Siz" (then they will align with the other attributes). But "Siz" is maybe a bit weird too, so I don't know what we should do. You can decide.

Reply:
Siz roughly corresponds to height in the game. 
"Wt " or "Wgt" would do

Reply:
I'm in favor of "Wgt" personally.

Reply:
Wgt looks weird to me, but according to Google both "wt" and "wgt" are common abbreviations.

What is "size" actually referring to?

If it's height then what about "Ht" and "Wt"?

Reply:
The size stat seems to mostly correlate to height, yes. But it's used more like a "how physically imposing is this thing" catch all stat so it's a bit inconsistent. For humanoids it does seem to be just intended as height in centimeters.

Reply:
I think I'll go with Ht and Wt then, unless there are strong objections. While it may be a bit strange for nonhumans, it is also nicely consistent and understandable.